src/benchmarks/micro/libraries/System.Numerics.Tensors/Perf_SignedIntegerTensorPrimitives.cs

```diff
@@ -0,0 +1,128 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Extensions;
+using MicroBenchmarks;
+using System.Linq;
+
+namespace System.Numerics.Tensors.Tests
+{
+    [BenchmarkCategory(Categories.Libraries, Categories.SIMD, Categories.JIT)]
+    [GenericTypeArguments(typeof(sbyte))]
```

Comment:
NB this configuration results in 756 total benchmarks being defined (which took over 1 hour to complete on my laptop). If it's too much I could try scaling back scope to more targeted type/API/input combinations.

Reply:
> If it's too much

I think that's probably too much :) @adamsitnik ? @DrewScoggins ?